packages/editor/src/components/Gutter/index.ts

```diff
@@ -0,0 +1,136 @@
+import type { Editor } from '@tiptap/core';
+import { consume } from '@lit/context';
+import numberBadgeStyles from '@nl-design-system-candidate/number-badge-css/number-badge.css?inline';
+import AccessibleIcon from '@tabler/icons/outline/accessible.svg?raw';
+import CloseIcon from '@tabler/icons/outline/x.svg?raw';
+import { html, LitElement, unsafeCSS } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { map } from 'lit/directives/map.js';
+import { createRef, ref, type Ref } from 'lit/directives/ref.js';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import { tiptapContext } from '@/context/tiptapContext.ts';
+import { type ValidationMeta, validationsContext, type ValidationsMap } from '@/context/validationsContext.ts';
+import gutterStyles from './styles.ts';
+
+@customElement('clippy-gutter')
+export class Gutter extends LitElement {
+  static override readonly styles = [gutterStyles, unsafeCSS(numberBadgeStyles)];
+  @state()
+  private isOpen = false;
+
+  @consume({ context: tiptapContext })
+  @property({ attribute: false })
+  public editor?: Editor;
+
+  @consume({ context: validationsContext, subscribe: true })
+  @property({ attribute: false })
+  validationsContext?: ValidationsMap;
+
+  private dialogRef: Ref<HTMLDialogElement> = createRef();
+
+  #toggleOpen = () => {
+    const { value } = this.dialogRef;
+    if (this.isOpen) {
+      value?.close();
+    } else {
+      value?.show();
+    }
+    this.isOpen = !this.isOpen;
+  };
+
+  #focusNode(item: ValidationMeta) {
+    try {
+      const { view } = this.editor || {};
+      const nodeDom = view?.nodeDOM?.(item.pos) ?? view?.domAtPos(item.pos).node;
+      if (nodeDom instanceof HTMLElement) {
+        nodeDom.scrollIntoView({ behavior: 'smooth', block: 'center' });
+      }
+
+      // Set a text selection at the position and focus the view
+      this.editor?.commands.focus(item.pos);
+      this.#toggleOpen();
+    } catch (err) {
+      console.error('Cannot scroll to and focus node', err);
+    }
+  }
+
+  renderValidationContent() {
```

Comment:
Dit voelt daarentegen misschien wel als een eigen component. Misschien een report (die in n modal kan renderen) en een indicator die in de gutter staat